corehq/apps/es/tests/test_command_make_elastic_migration.py

```diff
@@ -138,10 +135,9 @@ def test_build_migration_fails_for_multiple_operations_on_the_same_index(self):
         deletes = [conflict_index]
         command = Command()
         command.empty = False
-        with self.assertRaises(CommandError) as mock:
+        prefix = f"Multiple operations for the same index ({conflict_index}):"
+        with self.assertRaisesRegex(CommandError, f"^{re.escape(prefix)}.*"):
```

Comment:
nit: `.*` is unnecessary
```suggestion
        with self.assertRaisesRegex(CommandError, f"^{re.escape(prefix)}"):
```
Also applies to other tests using the same pattern.

Reply:
See 910aad67775